README.md

```diff
@@ -1 +1,178 @@
-# recipe-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/raw/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 RECIPE APP WITH ROR <a name="about-project"></a>
+
+> The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The following tech stack was used:
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+    <li><a href="https://reactjs.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Puma</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> The Following is how you can use the Blog App
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+> Ruby version : `ruby 3.1.2p20`
+    run `ruby -v`
+
+> SQLite3 version : `3.40.0`
+    run `sqlite3 --version`
+
+> npm version : `8.15.0`
+    run `npm -v`
+
+> yarn version : `1.22.19`
+    run `yarn -v`
+
+> Database : **PostgreSQL**
+### Setup
+
+Clone this repository to your desired folder:
+
+> `git clone git@github.com:David-Kasilia/blog_app_ror.git `
+
+### Install
+
+Install this project with:
+
+> `bundler install`
+
+### Usage
+
+To run the project, execute the following command:
+
+`rails server` or `rails s` 
```

Comment:
- [ ] Please include here that you need to run `rails assets:precompile` before `rails s` to be able to run the project.